statstracker/src/tests/App.test.js

```diff
@@ -0,0 +1,117 @@
+// Required to get <App /> running:
+import React from 'react';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import rootReducer from '../reducers/reducers';
+import App from '../components/App';
+const store = createStore(rootReducer);
+
+import { render, fireEvent } from 'react-testing-library';
+import 'jest-dom/extend-expect';
+
+const packagedApp = (
+  <Provider store={store}>
+    <App />
+  </Provider>
+);
+
+const { getByText } = render(packagedApp);
+
+// I think this is a pretty good way to click different buttons any number of times:
+const click = (button, number = 1) => {
+  // Wow! I finally used RegExp in a real project. This was hard for me to figure out! Lots of trial and error and reading...
+  const regex = new RegExp(button.concat('$'), 'i');
+  for (let i = 0; i < number; i += 1) {
+    fireEvent.click(getByText(regex));
+  }
+};
+
+beforeEach(() => {
+  // This is my goofy way of reseting for each test. I created an action, reducer, and button ('clear') to return all counts to 0. The button is styled { display: none }, so the user cannot see it or click it, but the test can use it. I know there is a better way, but I haven't found it yet.
+  click('clear');
+});
```

Comment:
hopefully our discussion in last night's standup and the extra research you did cleared this up